en/docs/Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -135,10 +135,10 @@ For example, let's share a JDBC user store (MySQL) with both the WSO2 Identity S
 
     **Example**
     
-    [![sample-sp](../../../assets/img/Learn/Extensions/SAML2SSO/sample-sp.png)](../../../assets/img/Learn/Extensions/SAML2SSO/sample-sp.png)
+    [![sample-sp]({{base_path}}/assets/img/Learn/Extensions/SAML2SSO/sample-sp.png)]({{base_path}}assets/img/Learn/Extensions/SAML2SSO/sample-sp.png)
```

Comment:
Path is incorrect
({{base_path}}assets/img/Learn/Extensions/SAML2SSO/sample-sp.png)
Add a / after {{base_path}}

Reply:
Fixed. thanks. https://github.com/wso2/docs-apim/pull/603/commits/42471e624336be179f2e5f22a8f5831ff70f8151